### ISSUE TITLE
0.59.0+ crashes on Image.from_registry

### ISSUE THREAD

**Issue by the-crypt-keeper** [modal-labs/modal-client#1467]
ENV: Python 3.10.12 Ubuntu-22 under WSL.

Up until 0.58.92 all is great ❤️  modal!

Anything after 0.59.0 chokes on `Image.from_registry` like so:

```
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /home/user/.local/bin/modal:8 in <module>                                                        │
│                                                                                                  │
│   7 │   sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])                         │
│ ❱ 8 │   sys.exit(main())                                                                         │
│   9                                                                                              │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal/__main__.py:18 in main                      │
│                                                                                                  │
│   17 │   except CliUserExecutionError as exc:                                                    │
│ ❱ 18 │   │   raise exc.__cause__ from None                                                       │
│   19 │   except Exception as exc:                                                                │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal/cli/import_refs.py:72 in                    │
│ import_file_or_module                                                                            │
│                                                                                                  │
│    71 │   │   try:                                                                               │
│ ❱  72 │   │   │   spec.loader.exec_module(module)                                                │
│    73 │   │   except Exception as exc:                                                           │
│ <frozen importlib._bootstrap_external>:883 in exec_module                                        │
│                                                                                                  │
│ <frozen importlib._bootstrap>:241 in _call_with_frames_removed                                   │
│                                                                                                  │
│                                                                                                  │
│ /home/user/work/ai/can-ai-code/interview_modal_cuda11.py:231 in <module>                         │
│                                                                                                  │
│   230 image = (                                                                                  │
│ ❱ 231 │   Image.from_registry("nvidia/cuda:11.8.0-devel-ubuntu22.04",                            │
│   232 │   │   │   │   │   │   setup_dockerfile_commands=["RUN apt-get update", "RUN apt-get in   │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/synchronicity/synchronizer.py:526 in proxy_method │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/synchronicity/synchronizer.py:421 in f_wrapped    │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal/image.py:1312 in run_function               │
│                                                                                                  │
│   1311 │   │                                                                                     │
│ ❱ 1312 │   │   function = _Function.from_args(                                                   │
│   1313 │   │   │   info,                                                                         │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal/functions.py:643 in from_args               │
│                                                                                                  │
│    642 │   │   │   if config.get("automount"):                                                   │
│ ❱  643 │   │   │   │   automounts = info.get_auto_mounts()                                       │
│    644 │   │   │   │   all_mounts += automounts                                                  │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal/_function_utils.py:267 in get_auto_mounts   │
│                                                                                                  │
│   266 │   │   │   │   potential_mount = _Mount.from_local_python_packages(module_name)           │
│ ❱ 267 │   │   │   │   mount_paths = potential_mount._top_level_paths()                           │
│   268 │   │   │   except ModuleNotMountable:                                                     │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal/mount.py:277 in _top_level_paths            │
│                                                                                                  │
│   276 │   │   for entry in self.entries:                                                         │
│ ❱ 277 │   │   │   res.extend(entry.top_level_paths())                                            │
│   278 │   │   return res                                                                         │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal/mount.py:212 in top_level_paths             │
│                                                                                                  │
│   211 │   │   paths = []                                                                         │
│ ❱ 212 │   │   for sub in self._proxy_entries():                                                  │
│   213 │   │   │   paths.extend(sub.top_level_paths())                                            │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal/mount.py:176 in _proxy_entries              │
│                                                                                                  │
│   175 │   def _proxy_entries(self) -> List[_MountEntry]:                                         │
│ ❱ 176 │   │   mount_infos = get_module_mount_info(self.module_name)                              │
│   177 │   │   entries = []                                                                       │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal_utils/package_utils.py:25 in                │
│ get_module_mount_info                                                                            │
│                                                                                                  │
│ /home/user/.local/lib/python3.10/site-packages/modal_utils/package_utils.py:14 in                │
│ get_file_formats                                                                                 │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: 'NoneType' object is not iterable
```

I have attempted to set `automount = false` in my .modal.toml to no effect.

**Comment by mwaskom** [comment]
Thanks for the report! Sounds like the same issue as #1451

**Comment by freider** [comment]
Sorry about this. Fixed in v0.61.4
